src/gui/tray/UserModel.cpp

```diff
@@ -643,14 +648,16 @@ Q_INVOKABLE void UserModel::openCurrentAccountLocalFolder()
 
 Q_INVOKABLE void UserModel::openCurrentAccountTalk()
 {
-    if (_users.isEmpty())
+    if (!currentUser())
         return;
 
-    QString url = _users[_currentUserId]->server(false) + "/apps/spreed";
-    if (!(url.contains("http://") || url.contains("https://"))) {
-        url = "https://" + _users[_currentUserId]->server(false) + "/apps/spreed";
+    auto talkApp = currentUser()->talkApp();
+    if (talkApp) {
+        QDesktopServices::openUrl(talkApp->url());
+    } else {
+        qCWarning(lcActivity) << "The Talk app is not enabled on " <<
```

Comment:
Somehow didn't notice it the previous time but you don't need the space before the closing double quote. They're added for you automatically, also you can probably have this on a single line.